op_crates/web/00_dom_exception.js

```diff
@@ -1,15 +1,47 @@
 // Copyright 2018-2020 the Deno authors. All rights reserved. MIT license.
 
-// eslint-disable-next-line @typescript-eslint/no-unused-vars
-class DOMException extends Error {
-  #name = "";
+((window) => {
+  const nameToCodeMapping = new Map([
+    ["IndexSizeError", 1],
+    ["HierarchyRequestError", 3],
+    ["WrongDocumentError", 4],
+    ["InvalidCharacterError", 5],
+    ["NoModificationAllowedError", 7],
+    ["NotFoundError", 8],
+    ["NotSupportedError", 9],
+    ["InvalidStateError", 11],
+    ["SyntaxError", 12],
+    ["InvalidModificationError", 13],
+    ["NamespaceError", 14],
+    ["InvalidAccessError", 15],
+    ["TypeMismatchError", 17],
+    ["SecurityError", 18],
+    ["NetworkError", 19],
+    ["AbortError", 20],
+    ["URLMismatchError", 21],
+    ["QuotaExceededError", 22],
+    ["TimeoutError", 23],
+    ["InvalidNodeTypeError", 24],
+    ["DataCloneError", 25],
+  ]);
+  class DOMException extends Error {
+    #name = "";
+    #code = 0;
```

Comment:
Nit: I don't like the redundancy of this with the `|| 0` part when setting `#code` in the constructor.
I'm tempted ro remove them both.